### DISCUSSION TITLE
PAGConvertor是否有linux版本？

### DISCUSSION THREAD

**Discussion by rebornfz** [Tencent/libpag#374] [Q&A] [answered]
[文档](https://pag.io/docs/animation-convertor.html)里只有mac版本的，是否有linux版本的PAGConvertor？

**Comment by kevingpqi123** [comment, marked as answer]
Linux 版本的 PAGConvertor 已增加，已上传官网：https://pag.io/docs/animation-convertor.html

**Reply by JonasTrinh** [reply]
网址已失效，官网上也并未找到linux版本，请问linux版本的PAGConvertor哪里能获取呢？

**Reply by kevingpqi123** [reply]
Linux 版本暂时没有办法提供给大家了，主要是开源协议的影响，Linux 版本使用到了 x264，只有替换后才能对外提供， mac 版本我们完成了替换

**Reply by FlamingJay** [reply]
请问，有没有什么途径编译出PAGConvertor么？

**Comment by Eldric-wd** [comment]
同问，能否提供linux版本

**Reply by kevingpqi123** [reply]
由于开源协议的问题，提供不了了，只有 mac 版的

**Comment by CodeJhF** [comment]
补充一下：Linux 版本已经重新提供下载，文档也已更新。

- Linux x86_64：https://pag.qq.com/file/PAGConvertor-linux.zip
- Linux arm64：https://pag.qq.com/file/PAGConvertor-linux-arm64.zip
- 下载页面：https://pag.io/docs/pag-convert-download.html

另外 macOS 版现在是 universal binary（原生支持 Apple Silicon）：
https://pag.qq.com/file/PAGConvertor-mac.zip
